tests/screenshot_gui.py

```diff
@@ -50,6 +50,26 @@ def runTest(self):
 
         print("Using proxy %s" % proxy)
         driver = lib.common.get_webproxy_driver(proxy)
+
+        return driver
+
+    def runTest(self):
+        driver = self.start_browser()
+
+        print("Browser is running, connect and debug")
+        print("Press Control-[ to exit interactive mode")
```

Comment:
isn't Control-] to exit interact mode?

Reply:
Will check that too before merging this...

Reply:
Fixed in commit